File: src/main/Java/org/team1085/robot/commands/Logging.java
package org.team1085.robot.commands;

import org.team1085.robot.Robot;

import edu.wpi.first.wpilibj.command.Command;

public class Logging extends Command {

	public Logging() {
		requires(Robot.dashboard);
	}

	protected boolean isFinished() {
		return false;
	}

	public void execute() {}

}

File: src/main/kotlin/org/team1085/robot/commands/Logging.kt
package org.team1085.robot.commands

import edu.wpi.first.wpilibj.command.Command
import org.team1085.robot.Robot

class Logging : Command () {

  init { requires(Robot.dashboard) }

  override fun isFinished () = false
  override fun execute () {
    
  }

}